src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -113,11 +116,12 @@ public function mkdir($dirs, $mode = 0777)
     /**
      * Checks the existence of files or directories.
      *
-     * @param string|iterable $files A filename, an array of files, or a \Traversable instance to check
+     * @param string|iterable $files  A filename, an array of files, or a \Traversable instance to check
+     * @param string|null     $ofType A file type to check against, skipped if null (backward compatibility)
```

Comment:
please remove the `(backward compatibility)` part. Allowing null is not only for BC (otherwise it would be a deprecated code path). It is a behavior we keep.

Reply:
Done